.babelrc

```diff
@@ -1,3 +1,11 @@
 {
-  "presets": ["es2015", "stage-2"]
+  "presets": ["stage-2", ["es2015", { "modules": false }]],
```

Comment:
This is necessary for the es-build. Tree shaking only works with es2015 import statements because they are statically structured unlike commonjs where you can require modules inside an if-statement for example.

For our commonjs build I did use the normal `es2015` preset scoped to the `BABEL_ENV=commonjs` environment variable. That way we can easily switch between different build outputs.

Originally i had put the `modules: false` setting under the `es` env-var, but somehow babel throws an exception there. This smells like a bug in the config parser of babel itself.